plenum/server/node.py

```diff
@@ -1928,9 +1922,8 @@ def sendLedgerStatus(self, nodeName: str, ledgerId: int):
         else:
             logger.info("{} not sending ledger {} status to {} as it is null".format(self, ledgerId, nodeName))
 
-    def send_ledger_status_to_client(self, lid, txn_s_n, v, p, merkle, protocol, client):
-        ls = LedgerStatus(lid, txn_s_n, v, p, merkle, protocol)
-        self.transmitToClient(ls, client)
+    def send_ledger_status_to_client(self, msg, client):
```

Comment:
I may be wrong, but I think that re-creation of a `LedgerStatus` was done explicitly in order to pass `CURRENT_PROTOCOL_VERSION` there. I think it wad important for some compatibility cases. Not sure if this is still a valid compatibility case, but I think it's worth double-checking.

Reply:
The problem exactly in recreation process. If there was an error with LedgerStatus schema (as in test i send a message with merkleRoot parameter as null not str) then creation LedgerStatus object back will cause the same error too. I think, that in this case we need to reply to client the same message with error describing.